source/app/views/forms.erb

```diff
@@ -0,0 +1,49 @@
+<script>
+  $(document).ready(function(){
+    $("#c_one").change(function(){
```

Comment:
Nice!

My only suggestion here would be to use more descriptive CSS IDs until this code reveals its meaning without examining the HTML.

    $(document).ready(function(){
        $("#player_one_throw").change(function(){
            $(".player_one_form").hide("slow");
            $(".player_two_form").show("slow");
        });
        $("#player_two_throw").change(function(){
          $(".player_two_form").hide("slow");
          $("#throw").show("slow");
        });
     });